src/Symfony/Component/Translation/CatalogueBag/ContainerAwareResourceCatalogueBag.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\CatalogueBag;
```

Comment:
Does that make sense in the `Translation` component? Shouldn't it be in the `FrameworkBundle` instead?